docs/voice-transformers/alien-voice/the-on-off-button.md

```diff
@@ -35,7 +36,7 @@ void HAL_GPIO_EXTI_Callback(uint16_t GPIO_Pin) {
 }
 ```
 
-The interrupt handler toggles the variable effect\_enabled and switches the LED on when its value is true.
```

Comment:
Not sure with this correction

Reply:
Oups my bad, I didn't follow the modification of variable name. It make's less sense IMO... Let's leave it as it is 👍